integration-tests/deploy.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+import os, argparse, sys, subprocess, shlex, docker,  requests, json
+from python_on_whales import docker as d_cli
+from docker import errors as d_err
+from docker import types as d_types
+from retry import *
+
+
+def request_metrics(port):
+    try:
+        return requests.get(f"http://0.0.0.0:{port}/metrics")
+    except requests.RequestException:
+        sys.exit('Failed to get backend status.')
+    
+         
+@retry((UserWarning), tries=5, delay=1, backoff=2, max_delay=5)
+def ensure_backend_up():
+    try:
+        for item in range(int(os.environ['BOB_NODES_AMOUNT'])):
```

Comment:
I suggest to pass min and max port as arguments and not use env variable, because this is the only place where it is needed and that behavior is not expected

Reply:
fixed